src/sql/base/browser/ui/taskbar/actionbar.ts

```diff
@@ -39,11 +41,16 @@ export class ActionBar extends ActionRunner implements IActionRunner {
 	// Elements
 	private _domNode: HTMLElement;
 	private _actionsList: HTMLElement;
+	private _overflow: HTMLElement;
+	private _moreItemElement: HTMLElement;
 
-	constructor(container: HTMLElement, options: IActionBarOptions = defaultOptions) {
+	private _collapseOverflow: boolean;
+
+	constructor(container: HTMLElement, options: IActionBarOptions = defaultOptions, collapseOverflow: boolean = false) {
```

Comment:
why are we not making this the default behavior?

Reply:
I asked Hannah about this and she said to make it opt-in for now, then to consider making it default behavior in future iterations based on customer feedback and the broader design strategy.